### DISCUSSION TITLE
Is there a way to switch a field to a different field type?

### DISCUSSION THREAD

**Discussion by diegovogel** [statamic/cms#7914] [Q&A] [answered]
Say I have a text field that has data in it for several entries. Then I decide that field needs to be a markdown field instead. Is there a way to change the field type of the existing field?

**Comment by jasonvarga** [comment, marked as answer]
Not through the UI. In the YAML file you can change the `type`, though.

**Reply by diegovogel** [reply]
Excellent, thanks!
